.github/workflows/build-and-push-docker.yml

```diff
@@ -0,0 +1,45 @@
+name: Docker build and push
+
+on:
+  push:
+    branches: ['v*.*.*']
+
+jobs:
+  docker:
+    name: docker build and push
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 🙂
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Create image and tag names 😊
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: terrestris/geoserver
+          tags: |
+            type=ref,event=branch
+          flavor: |
```

Comment:
i think this can still be removed

Reply:
Yes, 
```
          flavor: |
            latest=auto
```
should be removed or kept here and removed from L43. Looks like it is created twice.

Reply:
removed it for now in f75dc6ad64177660682d1a9e63e33936d72dc1ae